internal/pkg/source/source_test.go

```diff
@@ -34,33 +30,27 @@ type testConfig struct {
 	sinkPattern      string
 }
 
-func (c *testConfig) IsSource(t types.Type) bool {
-	d := utils.Dereference(t)
-	_, ok := d.(*types.Named)
-	if !ok {
-		return false
-	}
-
-	match, _ := regexp.MatchString(c.sourcePattern, d.String())
+func (c *testConfig) IsSourceType(path, typename string) bool {
+	match, _ := regexp.MatchString(c.sourcePattern, typename)
 	return match
 }
 
-func (c *testConfig) IsSanitizer(call *ssa.Call) bool {
-	match, _ := regexp.MatchString(c.sanitizerPattern, call.String())
+func (c *testConfig) IsSanitizer(p, r, n string) bool {
```

Comment:
```suggestion
func (c *testConfig) IsSanitizer(path, recv, name string) bool {
```
These names feel a bit too concise, and they are inconsistent with the other functions' signatures.